13_execution_time.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Implementation for RubyMonk Primer Problem 13 Time to run code
+# Problem Statement:
+# You are given some code in the form of lambdas.
+# Measure and return the time taken to execute that code.
+class ExecutionTime
+  def exec_time(proc)
```

Comment:
Not to be a jerk, but technically `procs` and `lambdas` are distinct things in ruby. :)

That being said, the differences between them won't matter here, but the naming might.

Here's a challenge for you, though -- can you accomplish the same thing by taking a code block instead of a proc/lambda object?

Reply:
Yeah, RubyMonk seems to often munge and interchange the terms `proc`, `lambda`, and `block`.  Words are hard, I guess, and this is an intro level course.  But I appreciate your calling out this distinction.

Now, regarding your challenge...
> can you accomplish the same thing by taking a code block instead of a proc/lambda object?

Do you mean using the implicit(ly passed) block, like this?...
```
def exec_time
  # your code here
  start_time = Time.now
  yield
  Time.now - start_time
end

Reply:
Not quite, but close.  I should clarify.  I challenge you to find a way to have the method take a code block as an argument, but still execute the code *as a proc* -- since that is the original requirement.  Effectively, you have to convert a code block to a proc.

As I said, purely an academic exercise.